src/main/java/org/kohsuke/github/GitHub.java

```diff
@@ -458,6 +463,7 @@ public boolean isCredentialValid() throws IOException {
             retrieve().to("/user", GHUser.class);
             return true;
         } catch (IOException e) {
+            logger.log(Level.FINEST, "Exception validating credentials on {0} with login {1}: {2}", new Object[]{this.apiUrl, this.login, e, e});
```

Comment:
:bug: you are trying to pass four parameters to a three-argument message parameter; probably you meant to put the last `e` outside the braces, but this does not work because the preformatted overloads do not also accept `Throwable`. Also including `Throwable.toString` in the message is pointless when you are including the full stack trace anyway. So what you meant was probably something like

```java
logger.log(Level.FINE, "Exception validating credentials on " + apiUrl + " with login " + login, e);
```